pkg/scalers/solr_scaler.go

```diff
@@ -0,0 +1,186 @@
+package scalers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	v2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+type solrScaler struct {
+	metricType v2.MetricTargetType
+	metadata   *solrMetadata
+	httpClient *http.Client
+	logger     logr.Logger
+}
+
+type solrMetadata struct {
+	host                       string
+	core                       string
```

Comment:
As per docs PR, wouldn't it be better to rename this to make it more end-user friendly?
```suggestion
	collection                 string
```

Reply:
Absolutely, I agree